lib/modules/apostrophe-workflow-modified-documents/views/manageListPage.html

```diff
@@ -12,13 +12,17 @@
     <td>{% if piece.workflowSubmitted %}{{ __('Yes') }}{% else %}{{ __('No') }}{% endif %}</td>
```

Comment:
This is duplicated in #258

Reply:
That branch has been merged and there are no conflicts, so resolving.